internal/cephfs/store/volumeoptions.go

```diff
@@ -235,6 +235,7 @@ func NewVolumeOptions(
 	opts.Monitors = strings.Join(clusterData.Monitors, ",")
 	opts.SubvolumeGroup = clusterData.CephFS.SubvolumeGroup
 	opts.Owner = k8s.GetOwner(volOptions)
+	opts.BackingSnapshot = true
```

Comment:
This will impact the volume creation for other access modes, the Request is to support backingSnapshot for only ROX modes. For other access modes, we should create a subvolume.